### PR TITLE
Feat: actions build and push vanilla ge image

### PR DESCRIPTION
**Pull request by Aylr**
## Caveats
- [ ] this is pushing to my dockerhub account so we need to change this to a BERA official one

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+#        with:
+#          ref: ${{ github.event.pull_request.head.ref }}
+#          fetch-depth: 0
+      - uses: docker/build-push-action@v1.1.0
+        with:
+          name: Build and push image
+          registry: registry.hub.docker.com
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: sctay/b-test
```
Comment (tomdavidson) [inline .github/workflows/docker-image.yml:24]: added you to beradev, but want to push this to gcr and/or github too

File: vanilla-ge/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+RUN apt-get update
+
+ADD ./requirements.txt /app/
+RUN pip install -r /app/requirements.txt
+WORKDIR /app
+
+LABEL maintainer="BERA"
+
+# Run great_expectations by default and allow any command to be run if desired.
+CMD ["great_expectations"]
```
Comment (tomdavidson) [inline vanilla-ge/Dockerfile:12]: I think this should be the entrypoint for the vanilla-ge

Comment (Aylr) [reply vanilla-ge/Dockerfile:12]: Agreed. Though sometimes I get lazy and just want to invoke a command in a container without the `--entrypoint` arg. 😄

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (Aylr) [reply .github/workflows/docker-image.yml:24]: Nice! Will do.

**Pull request merged**